tests/register.spec.cjs

```diff
@@ -0,0 +1,72 @@
+import { test, expect } from "@playwright/test";
+import { MongoClient } from "mongodb";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+let db;
+let client;
+
+test.beforeAll(async () => {
+	// @ts-ignore
+	client = new MongoClient(process.env.MONGO_URL);
+	await client.connect();
+	db = client.db("ecommerce-test");
+});
+
+let email = generateRandomEmail();
+
+function generateRandomEmail() {
```

Comment:
Same - might help avoid redundancy too

Reply:
I agree with you. Same reason as above should more files require this random email function, I will abstract it into a helper function folder in subsequent commits.